src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -25,19 +26,112 @@ import housing from '../../../../services/housing';
 import user from '../../../../services/user';
 const MAX_NUM_APPLICANTS = 8;
 
-const InstructionsCard = () => (
-  <Card>
-    <CardHeader title="Apartment Application Instructions" className="apartment-card-header" />
-    <CardContent>
-      <Typography variant="body1">Placeholder Text</Typography>
-      <Typography variant="body1">
-        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde
-        suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam
-        dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.
-      </Typography>
-    </CardContent>
-  </Card>
-);
+const InstructionsCard = () => {
+  const [apartmentSelectionDate, setApartmentSelectionDate] = useState();
+  const [thisYear, setThisYear] = useState();
+
+  useEffect(() => {
+    const getYear = () => setThisYear(new Date().getFullYear());
+
+    getYear();
+
+    // let selectionDate = housing.getApartmentSelectionDate();
+    let selectionDate = 'Apr. 27'; // API endpoint for this is planned for the future
+    setApartmentSelectionDate(selectionDate);
+  }, []);
+
+  const rows = [
+    { description: 'Current Freshman', points: 1 },
+    { description: 'Current Sophomore', points: 2 },
+    { description: 'Current Junior', points: 3 },
+    { description: 'Current Senior', points: 4 },
+    { description: '23+ years old', points: 1 },
+    { description: 'Full-time, off-campus program credit', points: 1 },
+    { description: 'Academic probation', points: -1 },
+    { description: 'Possible academic suspension', points: -2 },
+    { description: `${thisYear - 1}-${thisYear} Disciplinary Probation`, points: -3 },
+  ];
+
+  return (
+    <Card>
+      <CardHeader
+        title="On-Campus Apartments"
+        subheader="Information and Guidelines"
+        className="apartment-card-header"
+      />
+      <CardContent>
+        <Typography variant="body1" paragraph>
+          Apartments provide an alternative to the traditional residence hall setting and offer a
+          unique community experience. To be eligible to live in an apartment, students must be at
+          least 20 years old as of Sept. 1, {thisYear} <strong>or</strong> a current junior or
```

Comment:
Should we make a note here that the program does not currently check if they are eligible so "just because you are successfully added does not mean you should be"? Or should we have Ethan put that in the email he sends to tell people about this?

Reply:
Well that just means that right now the application is not different that the excel file (in regards to the eligibility), so I don't think there is any need to point that out differently than the instructions from Ethan